llarp/config/config.cpp

```diff
@@ -740,7 +740,7 @@ namespace llarp
 #endif
 
     // Default, but if we get any upstream (including upstream=, i.e. empty string) we clear it
-    constexpr Default DefaultUpstreamDNS{"1.1.1.1"};
+    constexpr Default DefaultUpstreamDNS{"9.9.9.9"};
```

Comment:
9.9.9.11 might be a better default.  `.9` says it blocks some things, and it isn't clear what.

Reply:
Oops, mixed it up.  I mean .10, not .11.